### DISCUSSION TITLE
Linking Images in Render Notes

### DISCUSSION THREAD

**Discussion by MrOctocat** [TriliumNext/Trilium#4763] [Q&A]
How to link an image from my Trilium database in a render note using the `<img />` tag? In other words, what should the value of the `src=""` attribute be if the image I want to link is not on the web but rather in my note database? Thank you.

EDIT:
I tried `<img src="api/images/{{NoteID}}" />`, where I replaced `{{NoteID}}` with the actual image note's ID value (no curly brackets), but the image didn't show up in the rendered note.

**Comment by Nriver** [comment]
I think you missed the last part of the url. It should be like this:
```
# Image type note
api/images/NOTE_ID/ANY_STRING
# Attachment
api/attachments/NOTE_ID/image/ANY_STRING
```

**Reply by MrOctocat** [reply]
Yes, that was it. Thank you so much.
